unlock-app/src/__tests__/components/lock/ShowUnlessUserHasKeyToAnyLock.test.js

```diff
@@ -49,8 +49,33 @@ describe('ShowUnlessUserHasKeyToAnyLock', () => {
       expect(wrapper.queryByText('Show me')).toBeNull()
     })
 
+    it('should show nothing if accounts are not loaded and 500ms has not elapsed (no flash)', () => {
```

Comment:
500ms or 200?

Reply:
yeah I was confused :) you can watch the confusion in action in the video I uploaded to google drive. Will fix